backend/app/contact_type/routes.py

```diff
@@ -15,3 +15,12 @@ def delete_contact_type(id):
     db.session.commit()
 
     return jsonify(contact_type.serialize)
+
+
+# get all contacts by contact_type
+@contact_type.route("/<uuid:id>/contacts", methods=["GET"])
+def get_all_contacts_by_contact_type(id):
+    contact_type = ContactType.query.filter_by(id=id).first()
```

Comment:
Could a non-existant id be passed in and cause errors ?

Reply:
Yes probably. Let me add a fix, and probably for the pr that was just merged as well lmfao.